modules/post-install-edge-aws-extend-cluster.adoc

```diff
@@ -1,21 +1,67 @@
 //
-// * post_installation_configuration/aws-compute-edge-tasks.adoc
+// * post_installation_configuration/aws-compute-edge-zone-tasks
 
 :_mod-docs-content-type: CONCEPT
 [id="post-install-edge-aws-extend-cluster_{context}"]
-= Extend existing clusters to use AWS Local Zones
+= Extend existing clusters to use AWS Local Zones or Wavelength Zones
 
-As a postinstallation task, you can extend an existing {product-title} cluster on Amazon Web Services (AWS) to use AWS Local Zones.
+As a post-installation task, you can extend an existing {product-title} cluster on Amazon Web Services (AWS) to use AWS {zone-type}.
 
-Extending nodes to Local Zone locations comprises the following steps:
-
-- Adjusting the cluster-network maximum transmission unit (MTU)
-- Opting in the Local Zone group to AWS Local Zones
-- Creating a subnet in the existing VPC for a Local Zone location
-- Creating the machine set manifest, and then creating a node in each Local Zone location
+Extending nodes to {zone-type} locations comprises the following steps:
 
+- Adjusting the cluster-network maximum transmission unit (MTU).
+- Opting in the {zone-type} group to AWS {zone-type}.
+- Creating a subnet in the existing VPC for a {zone-type} location.
++
 [IMPORTANT]
 ====
-Before you extend an existing {product-title} cluster on AWS to use Local Zones, check that the existing VPC contains available Classless Inter-Domain Routing (CIDR) blocks. These blocks are needed for creating the subnets.
+Before you extend an existing {product-title} cluster on AWS to use {zone-type}, check that the existing VPC contains available Classless Inter-Domain Routing (CIDR) blocks. These blocks are needed for creating the subnets.
 ====
++
+- Creating the machine set manifest, and then creating a node in each Local Zone or Wavelength Zone location.
+- Wavelength Zone only: Adding a permission to the Identity and Access Management (IAM) user or role, so that the required network resources can be created. For example:
++
+.Example of an additional IAM policy that attaches `ec2:ModifyAvailabilityZoneGroup`, `ec2:CreateCarrierGateway`, and `ec2:DeleteCarrierGateway` permissions to a user or role
++
+[source,yaml]
+----
+{
+  "Version": "2012-10-17",
+  "Statement": [
+    {
+      "Action": [
+        "ec2:ModifyAvailabilityZoneGroup"
+      ],
+      "Effect": "Allow",
+      "Resource": "*"
+    }
+  ]
+}
+----
++
+.Example of an additional IAM policy that attaches `ec2:ModifyAvailabilityZoneGroup`, `ec2:CreateCarrierGateway`, and `ec2:DeleteCarrierGateway` permissions to a user or role for a Wavelength Zone
++
+[source,yaml]
+----
+{
+  "Version": "2012-10-17",
+  "Statement": [
+    {
+      "Effect": "Allow",
+      "Action": [
+        "ec2:DeleteCarrierGateway",
+        "ec2:CreateCarrierGateway"
+      ],
+      "Resource": "*"
+    },
+    {
+      "Action": [
+        "ec2:ModifyAvailabilityZoneGroup"
+      ],
+      "Effect": "Allow",
+      "Resource": "*"
+    }
+  ]
+}
+----
```

Comment:
A little bit confused statement. Required to change before GA.

- We are providing an example without providing the instruction. Which permissions should be added?
- Missing for LZ
- Duplicate example description, different snippet

Suggestion:
```suggestion
- Local Zones only: Adding the permission  `ec2:ModifyAvailabilityZoneGroup` to the Identity and Access Management (IAM) user or role, so that the required network resources can be created. For example:
+
.Example of an additional IAM policy for AWS Local Zones deployments
+
[source,yaml]
----
{
  "Version": "2012-10-17",
  "Statement": [
    {
      "Action": [
        "ec2:ModifyAvailabilityZoneGroup"
      ],
      "Effect": "Allow",
      "Resource": "*"
    }
  ]
}
----


- Wavelength Zone only: Adding the permissions  `ec2:ModifyAvailabilityZoneGroup`, `ec2:CreateCarrierGateway`, and `ec2:DeleteCarrierGateway` to the Identity and Access Management (IAM) user or role, so that the required network resources can be created. For example:
+
.Example of an additional IAM policy for AWS Wavelength Zones deployments
+
[source,yaml]
----
{
  "Version": "2012-10-17",
  "Statement": [
    {
      "Effect": "Allow",
      "Action": [
        "ec2:DeleteCarrierGateway",
        "ec2:CreateCarrierGateway"
      ],
      "Resource": "*"
    },
    {
      "Action": [
        "ec2:ModifyAvailabilityZoneGroup"
      ],
      "Effect": "Allow",
      "Resource": "*"
    }
  ]
}
----
```

Reply:
Replaced